torchvision/transforms/functional.py

```diff
@@ -124,7 +125,7 @@ def _is_numpy_image(img: Any) -> bool:
     return img.ndim in {2, 3}
 
 
-def to_tensor(pic: Union[PILImage, np.ndarray]) -> Tensor:
+def to_tensor(pic: Union[PILImage, NDArray[np.uint8]]) -> Tensor:
```

Comment:
Sorry @adamtheturtle I don't think this is correct, `to_tensor` also accepts  numpy arrays of float values.

Reply:
Thank you for taking another look @NicolasHug . I have attempted to update this PR. We will get there!